src/components/left-nav-panel.tsx

```diff
@@ -16,21 +16,21 @@ export class LeftNavPanelComponent extends BaseComponent<IProps, IState> {
     return (
       <div className="left-nav-panel">
```

Comment:
We'll also want to add the legend from this PR:
https://github.com/concord-consortium/connected-bio-spaces/pull/4